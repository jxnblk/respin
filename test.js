const test = require('ava')
const React = require('react')
const render = require('react-test-renderer').create
const Spinner = require('./Spinner')

test('snapshot', t => {
  const tree = render(<Spinner />).toJSON()
  t.snapshot(tree)
})
