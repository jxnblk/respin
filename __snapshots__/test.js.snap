// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<svg
  fill="currentcolor"
  height={16}
  style={
    Object {
      "display": "inline-flex",
      "margin": 0,
    }
  }
  viewBox="0 0 32 32"
  width={16}
>
  <path
    d="M14 0 H18 V8 H14z"
    opacity={0.1}
    transform="rotate(0 16 16)"
  >
    <animate
      attributeName="opacity"
      begin="0ms"
      dur="1000ms"
      from={1}
      repeatCount="indefinite"
      to={0.1}
    />
  </path>
  <path
    d="M14 0 H18 V8 H14z"
    opacity={0.1}
    transform="rotate(45 16 16)"
  >
    <animate
      attributeName="opacity"
      begin="125ms"
      dur="1000ms"
      from={1}
      repeatCount="indefinite"
      to={0.1}
    />
  </path>
  <path
    d="M14 0 H18 V8 H14z"
    opacity={0.1}
    transform="rotate(90 16 16)"
  >
    <animate
      attributeName="opacity"
      begin="250ms"
      dur="1000ms"
      from={1}
      repeatCount="indefinite"
      to={0.1}
    />
  </path>
  <path
    d="M14 0 H18 V8 H14z"
    opacity={0.1}
    transform="rotate(135 16 16)"
  >
    <animate
      attributeName="opacity"
      begin="375ms"
      dur="1000ms"
      from={1}
      repeatCount="indefinite"
      to={0.1}
    />
  </path>
  <path
    d="M14 0 H18 V8 H14z"
    opacity={0.1}
    transform="rotate(180 16 16)"
  >
    <animate
      attributeName="opacity"
      begin="500ms"
      dur="1000ms"
      from={1}
      repeatCount="indefinite"
      to={0.1}
    />
  </path>
  <path
    d="M14 0 H18 V8 H14z"
    opacity={0.1}
    transform="rotate(225 16 16)"
  >
    <animate
      attributeName="opacity"
      begin="625ms"
      dur="1000ms"
      from={1}
      repeatCount="indefinite"
      to={0.1}
    />
  </path>
  <path
    d="M14 0 H18 V8 H14z"
    opacity={0.1}
    transform="rotate(270 16 16)"
  >
    <animate
      attributeName="opacity"
      begin="750ms"
      dur="1000ms"
      from={1}
      repeatCount="indefinite"
      to={0.1}
    />
  </path>
  <path
    d="M14 0 H18 V8 H14z"
    opacity={0.1}
    transform="rotate(315 16 16)"
  >
    <animate
      attributeName="opacity"
      begin="875ms"
      dur="1000ms"
      from={1}
      repeatCount="indefinite"
      to={0.1}
    />
  </path>
</svg>
`;
